drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1115,6 +1115,7 @@ function DragAndDropBlock(runtime, element, configuration) {
 
             if (attemptsRemain()) {
                 data.misplaced_items.forEach(function(misplaced_item_id) {
+                    state.last_action_correct = false
```

Comment:
@haikuginger @itsjeyd @bradenmacdonald I've removed this line and replaced it with a [proper fix](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/93/commits/042c56724ee3db06d4ea9eb0af2425182d79dc3c) - in two words, the reason behind this was that self.item_state was manipulated before value for `correct` was calculated.

So, since this is not the first time I see this problem (expected invocation order violation) in DnDv2 codebase, I've added [two TODOs](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/93/commits/2c78472bd5fbb92ac7ed354ad1513f0c826e4005) to refactor it to the methods that would benefit from such refactoring most. It doesn't look like a good time to try to address them now, so I've just added TODOs.

@bradenmacdonald let me know if you think we should create tickets for those TODOs.